packages/core/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { HubSession } from "./HubSession";
+export * from "./utils";
```

Comment:
Do we want to be more specific w/ our exports here?

Reply:
Sure - I had some vague recollection from @tomwayson about moving the super simple array/object util fns here for some tree shaking reasons... if we're not doing that, we can drop the second line